universum/configuration_support.py

```diff
@@ -391,6 +396,9 @@ def __add__(self, other: 'Step') -> 'Step':
             pass_tag=self.pass_tag + other.pass_tag,
             fail_tag=self.fail_tag + other.fail_tag,
             if_env_set=self.if_env_set + other.if_env_set,
+            # FIXME: This is a dummy implementation. Define addition logic and implement it.
```

Comment:
Please add the description of possible variants of combining conditional steps with groups/child steps to some task.

Reply:
Multiple high-level work items (alongside with the task you mentioned) were recorded in the corresponding [task](https://github.com/Samsung/Universum/issues/709). I would like to investigate possible combining variants later when I will start with that work item.